lightningd/plugin.c

```diff
@@ -1300,6 +1359,31 @@ void plugins_init(struct plugins *plugins)
 	plugins->default_dir = path_join(plugins, plugins->ld->config_basedir, "plugins");
 	plugins_add_default_dir(plugins);
 
+	/* Check that we found plugins that match the ones listed as
+	 * important.  */
+	for (size_t i = 0; i < tal_count(plugins->important); ++i) {
+		const char *name = plugins->important[i];
+		bool found = false;
+		struct plugin *p;
+
+		list_for_each(&plugins->plugins, p, list) {
+			if (plugin_paths_match(p->cmd, name)) {
+				found = true;
+				break;
+			}
+		}
+
+		if (!found) {
+			log_broken(plugins->log,
+				   "An --important-plugin=%s was not found, "
```

Comment:
```suggestion
				   "--important-plugin=%s was not found, "
```

Since we're giving the plugin name, no need to articalize it.